src/studio/src/designer/frontend/ux-editor/features/thirdPartyComponents/thirdPartyComponentSlice.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface IThirdPartyComponentsState {
+  components: any;
+  error: Error;
+}
+
+const initialState: IThirdPartyComponentsState = {
```

Comment:
Im unsure if this third party slice is needed really. Does this work at all anymore?

Reply:
I don't think it is in use at least. Might as well remove it.

Reply:
Removed any references to thirdpartycomponents.